apps/src/codebridge/FileBrowser/hooks/usePrompts.ts

```diff
@@ -24,7 +33,20 @@ export const usePrompts = () => {
     newFolder,
     projectFolders: project.folders,
     sendCodebridgeAnalyticsEvent,
-  });
+  } satisfies PAFunctionArgs<typeof globalOpenNewFolderPrompt>);
+
+  const openNewFilePrompt = usePartialApply(globalOpenNewFilePrompt, {
+    appName,
+    dialogControl,
+    newFile,
+    projectFiles: project.files,
+    sendCodebridgeAnalyticsEvent,
+    isStartMode,
+    validationFile,
+  } satisfies PAFunctionArgs<typeof globalOpenNewFilePrompt>);
 
-  return useMemo(() => ({openNewFolderPrompt}), [openNewFolderPrompt]);
+  return useMemo(
+    () => ({openNewFilePrompt, openNewFolderPrompt}),
+    [openNewFilePrompt, openNewFolderPrompt]
+  );
```

Comment:
The update here just adds in the partially applied `openNewFilePrompt` + all of the support arguments it needs.